readme.md

```diff
@@ -1,39 +1,38 @@
 # Laravel AutoDoc plugin 
+<p align="left">
+<a href="https://packagist.org/packages/ronasit/laravel-swagger"><img src="https://img.shields.io/packagist/dt/ronasit/laravel-swagger" alt="Total Downloads"></a>
+<a href="https://packagist.org/packages/ronasit/laravel-swagger"><img src="https://img.shields.io/packagist/v/ronasit/laravel-swagger" alt="Latest Stable Version"></a>
+<a href="https://packagist.org/packages/ronasit/laravel-swagger"><img src="https://img.shields.io/packagist/l/ronasit/laravel-swagger" alt="License"></a>
+</p>
 
-This plugin is designed to gather information and generate documentation about 
-your Rest-Api while passing the tests. The principle of operation is based on 
-the fact that the special Middleware installed on the Route for which you want 
-to collect information that after the successful completion of all tests 
-generated Swagger-file. In addition this plug-in is able to draw Swagger-template 
-to display the generated documentation for a config.
-
+## Introduction
+This plugin is designed to generate documentation about your Rest API while passing the tests. Special Middleware installed on the Route generates Swagger-file after the successful completion of all tests. In addition this plugin is able to draw Swagger-template to display the generated documentation for a config.
```

Comment:
could you please split the sentence into several lines so that one line contains a maximum of 80 characters?